roles/mirror_ocp_release/tasks/registry.yml

```diff
@@ -16,12 +19,19 @@
     --from={{ ocp_release_data['container_image'] | quote }}
     --to-release-image={{ mor_registry_url }}/{{ mor_registry_path }}:{{ mor_version }}
     --to={{ mor_registry_url }}/{{ mor_registry_path }}
+    {%- if mor_allow_insecure_registry | bool %}
+    --insecure
+    {%- endif %}
   retries: 3
   delay: 10
   register: _mor_result
   until: _mor_result.rc == 0
   changed_when: _mor_result.rc == 0
-  when: _mor_release_image.rc != 0 or mor_force | bool
+  when:
```

Comment:
Shouldn't this be something like...?

```
when: (_mor_release_image is defined and _mor_release_image.rc is defined and _mor_release_image.rc != 0) or mor_force | bool
```

Because it's a bit strange for me that we're mixing `_mor_release_image and mor_force in the same check when we have more extra checks for only `_mor_release_image`

Reply:
That looks good, will add it in next patch. Thanks